fulp_modules/bloodsuckers/bloodsucker/bloodsucker_coffin.dm

```diff
@@ -0,0 +1,176 @@
+	to_chat(owner, "<span class='announce'>Bloodsucker Tip: Find new lair recipes in the Misc tab of the <i>Crafting Menu</i>, including the <i>Persuasion Rack</i> for converting crew into Vassals.</span><br><br>")
+	RunLair() // Start
+	return TRUE
+
+// crate.dm
+/obj/structure/closet/crate
+	var/mob/living/resident	// This lets bloodsuckers claim any "closet" as a Coffin, so long as they could get into it and close it. This locks it in place, too.
+	var/pryLidTimer = 250
+	can_weld_shut = FALSE
+	breakout_time = 200
+
+/obj/structure/closet/crate/coffin/blackcoffin
+	name = "black coffin"
+	desc = "For those departed who are not so dear."
+	icon_state = "coffin"
+	icon = 'fulp_modules/bloodsuckers/icons/vamp_obj.dmi'
+	open_sound = 'fulp_modules/bloodsuckers/sounds/coffin_open.ogg'
+	close_sound = 'fulp_modules/bloodsuckers/sounds/coffin_close.ogg'
+	breakout_time = 600
+	pryLidTimer = 400
+	resistance_flags = NONE
+	material_drop = /obj/item/stack/sheet/metal
+	material_drop_amount = 2
+	armor = list("melee" = 50, "bullet" = 20, "laser" = 30, "energy" = 0, "bomb" = 50, "bio" = 0, "rad" = 0, "fire" = 70, "acid" = 60)
+
+/obj/structure/closet/crate/coffin/meatcoffin
+	name = "meat coffin"
+	desc = "When you're ready to meat your maker, the steaks can never be too high."
+	icon_state = "meatcoffin"
+	icon = 'fulp_modules/bloodsuckers/icons/vamp_obj.dmi'
+	resistance_flags = FIRE_PROOF
+	open_sound = 'sound/effects/footstep/slime1.ogg'
+	close_sound = 'sound/effects/footstep/slime1.ogg'
+	breakout_time = 200
+	pryLidTimer = 200
+	resistance_flags = NONE
+	material_drop = /obj/item/food/meat/slab/human
+	material_drop_amount = 3
+	armor = list("melee" = 70, "bullet" = 10, "laser" = 10, "energy" = 0, "bomb" = 70, "bio" = 0, "rad" = 0, "fire" = 70, "acid" = 100)
+
+/obj/structure/closet/crate/coffin/metalcoffin
+	name = "metal coffin"
+	desc = "A big metal sardine can inside of another big metal sardine can, in space."
+	icon_state = "metalcoffin"
+	icon = 'fulp_modules/bloodsuckers/icons/vamp_obj.dmi'
+	resistance_flags = FIRE_PROOF | LAVA_PROOF
+	open_sound = 'sound/effects/pressureplate.ogg'
+	close_sound = 'sound/effects/pressureplate.ogg'
+	breakout_time = 300
+	pryLidTimer = 200
+	material_drop = /obj/item/stack/sheet/metal
+	material_drop_amount = 5
+	armor = list("melee" = 40, "bullet" = 15, "laser" = 50, "energy" = 0, "bomb" = 10, "bio" = 0, "rad" = 0, "fire" = 70, "acid" = 60)
+
+//////////////////////////////////////////////
+
+/obj/structure/closet/crate/proc/ClaimCoffin(mob/living/claimant) // NOTE: This can be any "closet" that you are resting AND inside of.
+	// Bloodsucker Claim
+	var/datum/antagonist/bloodsucker/bloodsuckerdatum = claimant.mind.has_antag_datum(/datum/antagonist/bloodsucker)
+	if(bloodsuckerdatum)
+		// Vamp Successfuly Claims Me?
+		if(bloodsuckerdatum.ClaimCoffin(src))
+			resident = claimant
+			anchored = 1					// No moving this
```

Comment:
aw yeah, time to anchor my cardboard box

Reply:
(check the state of the anchorable variable before anchoring the closet)

Reply:
I mean, I guess cardboard boxes are subtypes of closets, but not crates, but you get my point